bfvmm/integration/arch/intel_x64/test_all.cpp

```diff
@@ -26,82 +26,30 @@
 //     from the EPT map.
 //
 
-#include <bfcallonce.h>
+#include <vmm.h>
```

Comment:
One change that I made is the `using namespace bfvmm::intel_x64` was moved to the include so that you don't have to do this manually.